src/components/strapi/index.test.ts

```diff
@@ -0,0 +1,42 @@
+import { createNodeCJSEnvironment } from "@kosko/env";
+
+import { create } from "./index";
+
+process.env.CI_COMMIT_SHA = "longsha";
+process.env.CI_COMMIT_SHORT_SHA = "shortsha";
+process.env.CI_ENVIRONMENT_SLUG = "some-slug";
+process.env.CI_ENVIRONMENT_NAME = "some-name";
+process.env.CI_PROJECT_PATH_SLUG = "some-path";
+process.env.KUBE_INGRESS_BASE_DOMAIN = "dev2.fabrique.social.gouv.fr";
+process.env.KUBE_NAMESPACE = "some-namespace";
```

Comment:
prefer mocking the `@socialgouv/kosko-charts/environment` inteface to not be github/gitlab spécifique

Reply:
un exemple ?